doc/tutorials/contributing-clion.md

```diff
@@ -115,49 +148,58 @@ the settings here:
 
-Make sure the selected "Scheme" is "Project".
+Make sure the selected _Scheme_ is _Project_.
 
 ## Development
 
 Usually the folders you have to work in to add functionality or documentation are
 as follows:
 
-- doc<br/>
+- **doc**<br/>
   This folder contains mainly all documentation of the project, including
   almost all pages of the [homepage](https://www.libelektra.org) of this
   project. One important note is, that all Markdown-pages can also be used
   for testing using
   [Markdown to Shell Recorder](https://github.com/ElektraInitiative/libelektra/tree/master/tests/shell/shell_recorder/tutorial_wrapper)
   (you can find an example on how to do this [here](/doc/help/kdb-get.md)).
-- src<br/>
+- **src**<br/>
   Almost all functionality-code resides here.
-  - bindings<br/>
+  - **bindings**<br/>
     Here is all the code of available [Bindings](/src/bindings/README.md)
     of libelektra.
-  - plugins<br/>
+  - **plugins**<br/>
     You can find all developed [Plugins](/src/plugins/README.md) here. If
     you want to fix a bug for an existing plugin or add another one, this is
     the directory you have to work in. For further information on how to
     develop your own plugin, please visit [this](/doc/tutorials/plugins.md)
     tutorial.
-  - tools<br/>
+  - **tools**<br/>
     In this folder the source of all tools for interacting with Elektra's
     global key database, e.g. _kdb_, can be found.
-- tests<br/>
+- **tests**<br/>
   Here you can find all sorts of tests (excluding tests created with the
   _Markdown to Shell Recorder_-tool).
 
 ## Testing
 
-The most thorough way to test your changes is to run all tests. Therefore
-navigate to your run-configurations (Run -> Edit Configurations...) and look
-for the entry `run_all`. There, `run_all` should be selected as `Executable`,
-`all` as `Target`. Now you can execute this run configuration which will run
-all enabled tests. Alternatively you can also run all tests using the terminal
-by executing `make run_all` inside your build folder (e.g. /cmake-build-debug).
+The most thorough way to test your changes is to run all tests.
+Therefore, navigate to your run-configurations (_Run_ --> _Edit Configurations..._)
+and look for the entry _CTest Application_ --> _All CTest_.
+The configuration should look like this:
+
+![CTest configuration](/doc/images/clion/ctest_config.png)
+
+You can easily run the tests by clicking on the icon right to the selected
+configuration:
+
+![CTest run](/doc/images/clion/ctest_run.png)
+
+Now you can execute this run configuration, which will run all enabled tests.
+Alternatively you can also run all tests using the terminal by executing `make run_all`
+inside your build folder (e.g. /cmake-build-debug).
 
 You can also run other specific tests by setting `Executable` to
-any of the `testmod_*` or `testkdb_*`targets. Additionally all tests using
+any of the `testmod_*` or `testkdb_*`targets. Additionally, all tests using
```

Comment:
Since version 2020.3 CLion also supports CTest tests, i.e. all other tests we have. I'm haven't used that feature and I'm not sure there is much benefit compared to just running test test executable, but the feature is there.